pkg/cmd/grafana-cli/commands/conflict_user_command.go

```diff
@@ -269,6 +288,18 @@ func getValidConflictUsers(r *ConflictResolver, b []byte) error {
 		// valid entry
 		newConflicts = append(newConflicts, *newUser)
 	}
+	for _, count := range counterKeepUsersForBlock {
+		// check if we only have one addition for each block
+		if count != 1 {
+			return fmt.Errorf("invalid number of users to keep, expected 1, got %d", count)
+		}
```

Comment:
thought: would it be worth printing which block has invalid number of users added/removed here and below as well?

Reply:
great thought! 💭